package.json

```diff
@@ -154,14 +154,13 @@
     "sass-loader": "13.3.2",
     "storybook": "7.1.0",
     "storybook-addon-apollo-client": "5.0.0",
-    "storybook-addon-launchdarkly": "2.0.0",
```

Comment:
We're not using this right now and with the update to `launchdarkly-react-client-sdk` 3.0.8 it caused the build to fail. Removing it fixed that. We can revisit later if we realize we do need this addon.